mailbox/memory/src/test/java/org/apache/james/mailbox/inmemory/JsoupTextExtractorTest.java

```diff
@@ -0,0 +1,27 @@
+package org.apache.james.mailbox.inmemory;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import org.apache.james.mailbox.extractor.TextExtractor;
+
+import java.io.InputStream;
+
+import org.junit.Before;
+import org.junit.Test;
+
+public class JsoupTextExtractorTest {
+    private TextExtractor textExtractor;
+
+    @Before
+    public void setUp() {
+        textExtractor = new JsoupTextExtractor();
+    }
+
+    @Test
+    public void HtmlTextTest() throws Exception {
+        InputStream inputStream = ClassLoader.getSystemResourceAsStream("documents/html.txt");
+        assertThat(inputStream).isNotNull();
+        assertThat(textExtractor.extractContent(inputStream, "text/html", null).getTextualContent())
```

Comment:
It's there for sure that the resource is existed. But to make only one assertion, I will remove it